static-build/qemu/build-static-qemu.sh

```diff
@@ -26,6 +26,9 @@ fi
 [ -n "$qemu_repo" ] || die "failed to get qemu repo"
 
 [ -n "$qemu_version" ] || qemu_version=$(get_from_kata_deps "assets.hypervisor.qemu.version")
+if ! (git ls-remote --heads "${qemu_url}" | grep -q "refs/heads/${qemu_version}"); then
```

Comment:
Your commit says 'use the tag instead of the version', but is this more 'if the version does not exist, try the tag' ?

Reply:
right, fixed commit message, thx.